### ISSUE TITLE
About Training Details.

### ISSUE THREAD

**Issue by NeverMoreLCH** [bertinetto/cfnet#34]
Hi,

Your work is really great! 
I have some questions when I train the cfnet-conv2 network. In your code 'run_experiment_cfnet_conv2.m', the epoch is set to 100. But in your pretrained networks, the pretrained networks for cfnet-conv2 are called 'cfnet-conv2_e80.mat' and 'cfnet-conv2_gray_e40.mat'. I want to know how many epochs do you set when training the cfnet-conv2 network (color and gray).

Thank you very much!
Hope your reply!

**Comment by bertinetto** [comment]
Hi,
This is the protocol we adopted, which we used to avoid overfitting to the test set(s).

* We train CFNet/SiamFC for 100 epochs
* We measure its *tracking* performance (on validation set) every 5 epochs (or 10, I can't remember).
* To report the results in plots (with error bars), we consider the variance of the last 10  (evaluated) epochs.
* To report final results and compare against the state-of-the-art, we pick the epoch of the network that maximizes the performance on the validation set and then run it on test set (e.g OTB/LaSoT/GOT-10k...). That epoch is reported in the mat file id (e.g. e40 or e80).

Hope this helps!
